packages/core-data/src/entities.js

```diff
@@ -31,6 +31,8 @@ export const rootEntitiesConfig = [
 				'site_icon_url',
 				'site_logo',
 				'timezone_string',
+				'defaultTemplatePartAreas',
+				'defaultTemplateTypes',
```

Comment:
Looks like these are the only fields that are camelCase? should they follow the other pattern?